### DISCUSSION TITLE
Once OCRmyPDF has created a PDFa file, how can I just output the OCRed text?

### DISCUSSION THREAD

**Discussion by reinaldocrespo** [ocrmypdf/OCRmyPDF#1338] [General]
As I understand from reading here, OCRmyPDF will take a regular PDF as input and outputs a searchable PDF (PDFa) by adding invisible text.   How would I create a simple .txt file with only the OCRed text from that newly created PDFa? 

Thank you.

**Comment by jbarlow83** [comment]
`ocrmypdf --sidecar ocrtext.txt in.pdf out.pdf` (for just OCR text)

Use `pdftotext` (from poppler-utils package) to get all text out, including text that is not from OCR
